test/parallel/test-cluster-worker-deprecated.js

```diff
@@ -7,12 +7,12 @@ const cluster = require('cluster');
 const worker = new cluster.Worker();
 
 assert.strictEqual(worker.exitedAfterDisconnect, undefined);
-assert.strictEqual(worker.suicide, undefined);
+assert.strictEqual(worker.exitedAfterDisconnect, undefined);
```

Comment:
Can this test just be removed? AIUI from https://github.com/nodejs/node/commit/3f6152108744d9c212f11603cfdaeb201f41e376 this is just to test that the deprecated properties work properly, and we're removing them.

cc/ @Trott